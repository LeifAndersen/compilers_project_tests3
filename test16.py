1 and 2 or 3 or not False and (True or 3 or 4 and (False or True) and False)

