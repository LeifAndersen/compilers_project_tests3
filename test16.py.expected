(program (or (and 1 2) 3 (and (not #f) (or #t 3 (and 4 (or #f #t) #f)))))

